magpie/api/requests.py

```diff
@@ -120,33 +119,33 @@ def get_user_matchdict_checked(request, user_name_key="user_name"):
     group = evaluate_call(lambda: zig.GroupService.by_group_name(group_name, db_session=request.db),
-                          fallback=lambda: request.db.rollback(), httpError=HTTPForbidden,
-                          msgOnFail=s.Group_MatchDictCheck_ForbiddenResponseSchema.description)
-    verify_param(group, notNone=True, httpError=HTTPNotFound,
-                 msgOnFail=s.Group_MatchDictCheck_NotFoundResponseSchema.description)
+                          fallback=lambda: request.db.rollback(), http_error=HTTPForbidden,
+                          msg_on_fail=s.Group_MatchDictCheck_ForbiddenResponseSchema.description)
+    verify_param(group, not_none=True, http_error=HTTPNotFound,
+                 msg_on_fail=s.Group_MatchDictCheck_NotFoundResponseSchema.description)
     return group
 
 
 def get_resource_matchdict_checked(request, resource_name_key="resource_id"):
     # type: (Request, Str) -> models.Resource
     resource_id = get_value_matchdict_checked(request, resource_name_key)
-    resource_id = evaluate_call(lambda: int(resource_id), httpError=HTTPBadRequest,
-                                msgOnFail=s.Resource_MatchDictCheck_BadRequestResponseSchema.description)
+    resource_id = evaluate_call(lambda: int(resource_id), http_error=HTTPBadRequest,
+                                msg_on_fail=s.Resource_MatchDictCheck_BadRequestResponseSchema.description)
     resource = evaluate_call(lambda: zig.ResourceService.by_resource_id(resource_id, db_session=request.db),
-                             fallback=lambda: request.db.rollback(), httpError=HTTPForbidden,
-                             msgOnFail=s.Resource_MatchDictCheck_ForbiddenResponseSchema.description)
-    verify_param(resource, notNone=True, httpError=HTTPNotFound,
-                 msgOnFail=s.Resource_MatchDictCheck_NotFoundResponseSchema.description)
+                             fallback=lambda: request.db.rollback(), http_error=HTTPForbidden,
+                             msg_on_fail=s.Resource_MatchDictCheck_ForbiddenResponseSchema.description)
+    verify_param(resource, not_none=True, http_error=HTTPNotFound,
+                 msg_on_fail=s.Resource_MatchDictCheck_NotFoundResponseSchema.description)
     return resource
 
 
 def get_service_matchdict_checked(request, service_name_key="service_name"):
     service_name = get_value_matchdict_checked(request, service_name_key)
     service = evaluate_call(lambda: models.Service.by_service_name(service_name, db_session=request.db),
-                            fallback=lambda: request.db.rollback(), httpError=HTTPForbidden,
-                            msgOnFail=s.Service_MatchDictCheck_ForbiddenResponseSchema.description)
-    verify_param(service, notNone=True, httpError=HTTPNotFound, content={u'service_name': service_name},
-                 msgOnFail=s.Service_MatchDictCheck_NotFoundResponseSchema.description)
+                            fallback=lambda: request.db.rollback(), http_error=HTTPForbidden,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lambda may not be necessary](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)